label_studio/tasks/functions.py

```diff
@@ -140,14 +140,13 @@ def fill_annotations_project():
     logger.info('Finished filling project field for Annotation model')
 
 
-def _fill_predictions_project(project_id):
-    Prediction.objects.filter(task__project_id=project_id).update(project_id=project_id)
+def _fill_predictions_project():
```

Comment:
I've cleaned this up as per our discussion so it doesn't spin up a different async process per project.